docs/Consensus/ibft/ibft-parameters.md

```diff
@@ -0,0 +1,75 @@
+# IBFT parameters
+
+## CLI options
+
+### Block period
+
+`--istanbul.blockperiod 1`
+
+Setting the block period is used for how long blocks should be minted by the validators. It is also used for validation
+of block times by all nodes, so should not be changed after deciding a value for the network.
+The setting is a positive integer, and measures the minimum numbers of seconds before the next block is considered 
+valid.
+
+The default value is `1`.
+
+### Request timeout
+
+`--istanbul.requesttimeout 10000`
+
+The request timeout is the timeout at which IBFT will seek to trigger a new round if the previous one did not complete.
+This period increases are the timeout is hit more often. This parameter sets the minimum timeout in the case of normal 
+operation and is measured in milliseconds.
+
+The default value is `10000`.
+
+## Genesis file options
+
+Within the `genesis.json` file, there is an area for IBFT specific configuration, much like a Clique network 
+configuration. 
+
+The options are as follows: 
+```
+{
+    "config": {
+        "istanbul": {
+            "epoch": 30000,
+            "policy": 0,
+            "ceil2Nby3Block": 0
```

Comment:
is not missing the example of `qibftBlock`?

Reply:
Done